### DISCUSSION TITLE
The Mariana Trench, the world's deepest, is closest to which of these US territories?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#467] [Galaxy Trivia] [answered]
**Hint:** 322km SW

**Comment by peterbaccaro** [comment]
Guam

**Comment by ralphcajipe** [comment, marked as answer]
The Mariana Trench is closest to Guam, a US territory. The US has jurisdiction over most of the trench because it is closest to Guam.  It is approximately 322 kilometers (200 miles) southwest of Guam. Guam is the largest and southernmost island of the Mariana Islands, and it serves as a strategic location for U.S. military operations in the region.

**Reply by ParanoidUser** [reply]
Bingo! One more for the *Galaxy Brain* badge. 😊
